recipes/omnisci-pytools/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "omnisci-pytools" %}
+{% set version = "0.9.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # url: https://github.com/Quansight/{{ name }}/archive/v{{ version }}.tar.gz
+  # sha256: a6d30fbe6787b4ed4e2049ff298900fa5ab9919601f4ada0eb6a26ce9cf1ad6e
+  git_url: https://github.com/gnestor/jupyterlab-omnisci.git
+  git_tag: no-running-ipython
```

Comment:
We don't allow building from git: https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos